kerykeion/charts/charts_utils.py

```
@@ -686,6 +686,28 @@ def draw_house_sectors(
+        negative_label: Label for negative direction (e.g., "S", "W").
+
+    Returns:
+        Formatted string (e.g., "52°7'25\" N").
+    """
+    sign = positive_label
+    if coord < 0.0:
+        sign = negative_label
+        coord = abs(coord)
+    deg = int(coord)
+    minutes = int((float(coord) - deg) * 60)
+    sec = int(round(float(((float(coord) - deg) * 60) - minutes) * 60.0))
```

Ruff (0.15.10)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)